ls1intum/Artemis
# `General`: Make feedback score editable when using assessment instructions

### Checklist
#### General


#### Client

### Motivation and Context
Closes #4515.

### Description
The grading scale of the assessment instructions suggests that tutors should be able to edit the score of the feedback when using assessment instructions.

### Steps for Testing
1. Start a manual assessment in an exercise with structured grading criteria.
2. Add a feedback. The feedback score should be editable, both before and after dragging an assessment instruction into it.

### Review Progress
#### Code Review
#### Manual Tests

### Test Coverage
unchanged

### Screenshots
c.f. Screenshots in #4515, points is now no longer disabled when an assessment instruction has been used.
